integrationtests/test_i18n.py

```diff
@@ -20,3 +53,88 @@ def test_js_english_interpolation(self):
         # Wait for page to load
         b.assert_element('input[name="name"][value="Untitled Workflow"]', wait=True)
         b.assert_element(".attribution .metadata", text="by Jane Doe")
+
+
+class TestLocaleSwitch(LocaleSwitcherIntegrationTest):
```

Comment:
No need for a new class. The other one has a buggy name -- it should be `class TestI18n(WorkbenchBase)`. And all the methods in this Python module should be within that one class.